docs/components.md

```diff
@@ -41,9 +41,8 @@ Ionic apps are made of high-level building blocks called Components, which allow
 </DocsCard>
 
 <DocsCard header="Card" href="api/card" icon="/icons/component-card-icon.png">
-  <p>
-    Cards are a great way to display an important piece of content, and can contain images, buttons, text, and more.
-  </p>
+  <!-- prettier-ignore -->
+  <p>Cards are a great way to display an important piece of content, and can contain images, buttons, text, and more.</p>
```

Comment:
MDX breaks when there are new lines within tags.

Docusaurus [recommends](https://docusaurus.io/docs/next/markdown-features/react) to either use `{/* prettier-ignore */}` or add the files to `.prettierignore` until Prettier has support for MDX 3.